src/test/data/JsonSerializableTutorsPetTest/Class/invalidClassTutorsPet.json

```diff
@@ -8,6 +8,7 @@
   } ],
   "classes" : [ {
     "name" : "CS2100 Lab",
-    "studentUuids" : [ null ]
+    "studentUuids" : [ null ],
+    "lessons" : [ ]
```

Comment:
Could we test for a `null` lesson as well? Perhaps in another file `nullLessonTutorsPet.json` and this file can be renamed `nullStudentUuidsTutorsPet.json`.